cmd/root.go

```diff
@@ -74,6 +74,13 @@ var rootCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		ketall.KetAll(ketallOptions)
 	},
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		// For Cobra's internal `__complete` command, errors must go to stderr.
+		// This command is used for command completion and sends debug messages on stderr.
+		if cmd.Name() == cobra.ShellCompRequestCmd {
+			cmd.Root().SetErr(ketallOptions.Streams.ErrOut)
```

Comment:
I think we should set it unconditionally in `Execute` below. Or do you think this could mess up something?

Reply:
Yes that would be simpler.  I just didn't know if you had set things towards stdout for some other reason.  I'll update the PR.  Thanks for the quick review!

> Thank you, Marc, for bringing this to Cobra and then taking care of small projects like this!

I enjoyed seeing that you had gotten rid of your own `completion` command and started to use Cobra's.  It always feels good to see people benefitting. 👍